core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -316,11 +316,10 @@ import com.datastax.driver.core.utils.UUIDs
 
         if (missing.deadline.isOverdue()) {
           log.info(
-            "[{}] {}: Failed to find missing sequence nr: {}. PersistenceId: {}",
+            "[{}] [{}]: Failed to find missing sequence nrs. Missing info: [{}]",
```

Comment:
So logging the complete `LookingForMissing` case class. Seems OK since this should only happen in error cases, right? Should we make it a warning or can this happen during normal operation as well?

Reply:
There is one edge case where it happens and it isn't an error. When we see a persistenceId for the first time and it isn't sequence nr 1 we look in the last two buckets for the previous sequence nrs and if they are found great, if not we assume that this is the earliest sequence nr we are going to see. 

If it is not that case the stage fails just below. For that reason, we could reduce this to DEBUG but I'd rather have a bit too much at Info until the new events by tag implementation stabilises a bit more.